src/Features/CSharp/Portable/Diagnostics/Analyzers/CSharpTypeStyleDiagnosticAnalyzerBase.cs

```diff
@@ -91,6 +92,19 @@ private void HandleVariableDeclaration(SyntaxNodeAnalysisContext context)
                     shouldAnalyze = IsStylePreferred(semanticModel, optionSet, state, cancellationToken);
                 }
             }
+            else if (declarationStatement.IsKind(SyntaxKind.DeclarationExpression))
+            {
+                var declaration = (DeclarationExpressionSyntax) declarationStatement;
+                declaredType = declaration.Type;
+
+                shouldAnalyze = ShouldAnalyzeDeclarationExpression(declaration, semanticModel, cancellationToken);
+
+                if (shouldAnalyze)
+                {
+                    state = State.Generate(declarationStatement, semanticModel, optionSet, isVariableDeclarationContext: false, cancellationToken: cancellationToken);
```

Comment:
Dumb question, is "isVariableDeclarationContext" necessary given that State.Generate can infer it from the first parameter passed in?